packages/editor-ui/src/components/NodeList.vue

```diff
@@ -86,44 +94,44 @@ export default mixins(genericHelpers).extend({
 </script>
 
 <style lang="scss" module>
-  .list {
-  	max-width: 100px;
-  	height: 20px;
-  	display: flex;
-  	flex-direction: row;
-  	justify-content: flex-end;
-  	align-items: center;
+.list {
+	max-width: 100px;
+	height: 20px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	align-items: center;
 
-  	.container {
-  		width: 18px;
-  		height: 18px;
-  		margin-left: var(--spacing-2xs);
-  		position: relative;
-  		display: block;
+	.container {
+		width: 18px;
+		height: 18px;
+		margin-left: var(--spacing-2xs);
+		position: relative;
+		display: block;
 
-  		.image {
-  			width: 18px;
-  			height: 18px;
-  			display: block;
-  		}
-  	}
+		.image {
+			width: 18px;
+			height: 18px;
+			display: block;
+		}
+	}
 
-  	.button {
-  		width: 20px;
-  		min-width: 20px;
-  		height: 20px;
-  		margin-left: var(--spacing-2xs);
-  		top: 0px;
-  		position: relative;
-  		display: flex;
-  		justify-content: center;
-  		align-items: center;
-  		background: var(--color-background-light);
-  		border: $--version-card-border;
-  		border-radius: var(--border-radius-base);
-  		font-size: 10px;
-  		font-weight: var(--font-weight-bold);
-  		color: var(--color-text-base);
-  	}
-  }
+	.button {
+		width: 20px;
+		min-width: 20px;
+		height: 20px;
+		margin-left: var(--spacing-2xs);
+		top: 0px;
+		position: relative;
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		background: var(--color-background-light);
+		border: $--version-card-border;
```

Comment:
I will update this. I know that I have not but this $--version-card-border here :)

Reply:
*Updated